unocss/mod.ts

```diff
@@ -74,21 +74,21 @@ export default (userOptions: DeepPartial<Options> = {}) => {
 
       // Generate the stylesheets for all pages
       site.processAll([".html"], async (pages) => {
-        const fileClassMap = new Map();
+        const pageClassMap = new Map<string, Set<string>>();
         const classes = new Set<string>();
 
         await Promise.all(
           pages.map(async (page) =>
-            fileClassMap.set(
-              page.src,
+            pageClassMap.set(
+              page.src.slug,
```

Comment:
`page.src.slug` is not unique per page. For example `/foo.md` and `docs/foo.md` have the same slug.
I think this map is unnecessary because there wont be duplicated pages here. I think you can add directly the content in the `classes` Set instance

Reply:
Okay, fixing now.